security/ldap.rst

```diff
@@ -268,6 +270,15 @@ implementation. Commonly used values are:
 If you pass ``null`` as the value of this option, the default UID key is used
 ``sAMAccountName``.
 
+extra_fields
+............
+
+**type**: ``array`` **default**: ``null``
+
+This is the custom fields to pull from the LDAP server. If the fields does not
```

Comment:
Defines the custom fields...

... If any field does not exist, an Invalid... will be thrown.